### ISSUE TITLE
[BUG] Invisible text on welcome widget

### ISSUE THREAD

**Issue by Krisseck** [paperless-ngx/paperless-ngx#4828]
### Description

![image](https://github.com/paperless-ngx/paperless-ngx/assets/471887/239fbcb9-cc8a-4784-b4c9-4401d59ae414)

I just downloaded and installed the project from Docker hub and this is on first time logging in. The welcome widget has invisible text in it.

I can "paint over" the text with my mouse and can see the text underneath.

I get the same problem with latest versions of Chrome and Firefox. 

### Steps to reproduce

1. Copy `[docker-compose.sqlite.yml](https://github.com/paperless-ngx/paperless-ngx/blob/main/docker/compose/docker-compose.sqlite.yml)`
2. Run said file with `docker compose up`
3. Login as superuser
4. Get the error

### Webserver logs

```bash
No logs related to this, seems to be purely frontend bug.
```

### Browser logs

```bash
No errors in console, no 404 assets etc.
```

### Paperless-ngx version

2.0.1

### Host OS

Arch linux

### Installation method

Docker - official image

### Browser

Chrome, Firefox

### Configuration changes

Changed port to 8064

**Comment by shamoon** [comment]
Ha, I dont use light mode ever, thanks, will sort that out.

**Comment by shamoon** [comment]
Closed in branch by #4829
